app/modules/inventory/views/templates/inventory_network_nav.hbs

```diff
@@ -10,6 +10,11 @@
       </div>
     </div>
     <div id="usersList"></div>
+
+    <div class="friendRequestsWrapper hidden">
+      <p class="list-label">{{I18n 'friend requests'}}</p>
```

Comment:
What about longer/more explicit text like : "Users who wants to become friends with you" or "Users requesting friendship"

Reply:
pending friend requests?